src/fides/api/service/connectors/saas_connector.py

```diff
@@ -456,16 +459,27 @@ def process_response_data(
         response_data: Union[List[Dict[str, Any]], Dict[str, Any]],
         identity_data: Dict[str, Any],
         postprocessors: Optional[List[PostProcessorStrategy]],
+        input_data: Optional[Dict[str, List[Any]]],
     ) -> List[Row]:
         """
         Runs the raw response through all available postprocessors for the request,
         forwarding the output of one postprocessor into the input of the next.
 
         The final result is returned as a list of processed objects.
         """
-
         rows: List[Row] = []
         processed_data = response_data
+        param_values: Dict[str, Any] = {}
+        if (
+            self.current_saas_request.param_values is not None
+            and input_data is not None
+        ):
+            for param_value in self.current_saas_request.param_values or []:
+                if param_value.references:
+                    input_list = input_data.get(param_value.name)
+                    if input_list:
+                        param_values[param_value.name] = input_list[0]
+
```

Comment:
We shouldn't need to do anything with `param_values`. The `configuration` for the postprocessor will have the `dataset_reference` value we care about.
```
postprocessors:
            - strategy: filter
              configuration:
                field: customerId
                value:
                  dataset_reference: <instance_fides_key>.customer.id
```

Then somewhere in the postprocessor filter, we'd do something like this
```
access_data.get(self.value.dataset_reference)
```
See the next comment for where `access_data` should come from

Reply:
for future reference. access_data.get(self.value.dataset_reference) did not work per se since the access_data has a special format, that being "<datatset:collection>.field" so i had to accomodate to that.